builder/azure/dtl/builder_acc_test.go

```diff
@@ -3,25 +3,16 @@
 
 package dtl
 
-// these tests require the following variables to be set,
-// although some test will only use a subset:
-//
+// Required environment variables
 // * ARM_CLIENT_ID
 // * ARM_CLIENT_SECRET
 // * ARM_SUBSCRIPTION_ID
-// * ARM_OBJECT_ID
+// * ARM_TENANT_ID
+// * ARM_RESOURCE_GROUP_NAME
 //
-// The subscription in question should have a resource group
-// called "packer-acceptance-test" in "South Central US" region.
-// This also requires a Devtest lab to be created with "packer-acceptance-test"
-// name in "South Central US region. This can be achieved using the following
-// az cli commands "
-// az group create --name packer-acceptance-test --location "South Central US"
-// az deployment group create \
-//  --name ExampleDeployment \
-//  --resource-group packer-acceptance-test \
-//  --template-file acceptancetest.json \
-
+// Your resource group should be the South Central US region
```

Comment:
Currently a few things are still hard coded, the region, the DTL lab name, the gallery names, and the gallery image names, all of these should be parameterized but ideally we would do this in a variables file instead of adding so many more environment variables, as this is already cumbersome, my goal is to wait for a released version of packer-plugin-sdk that contains the ability to run acceptance tests with extra args, so that I can use a variable file here with preset defaults, and then individual contributors change it, and CI can have its own full set up too.